src/pages/about.vue

```diff
@@ -3,88 +3,82 @@
     <div class="container">
       <div>
         <h1 class="text-5xl mb-10">
-          {{ $t('about.title') }}
+          {{ $t('about.title', { openverse: 'Openverse' }) }}
         </h1>
         <div class="content">
-          <p>{{ $t('about.description.content') }}</p>
+          <p>
+            {{ $t('about.description.content', { openverse: 'Openverse' }) }}
+          </p>
 
           <i18n path="about.collection.content" tag="p">
+            <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+            <template #openverse>Openverse</template>
             <template #common-crawl>
-              <VLink
-                :aria-label="$t('about.aria.common-crawl')"
-                href="http://commoncrawl.org/"
-                >{{ $t('about.collection.common-crawl') }}</VLink
-              >
+              <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+              <VLink href="http://commoncrawl.org/">Common Crawl</VLink>
             </template>
           </i18n>
 
           <i18n path="about.planning.content" tag="p">
+            <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+            <template #openverse>Openverse</template>
             <template #meta>
-              <VLink :aria-label="$t('about.aria.meta')" href="/meta-search">{{
-                $t('about.planning.meta')
-              }}</VLink>
+              <VLink href="/meta-search">{{ $t('about.planning.meta') }}</VLink>
             </template>
             <template #search>
-              <VLink
-                :aria-label="$t('about.aria.search')"
-                href="https://github.com/wordpress/openverse-frontend/"
-                >{{ $t('about.planning.search') }}</VLink
-              >
+              <!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
+              <VLink href="https://github.com/wordpress/openverse-frontend/">{{
+                $t('about.planning.meta')
+              }}</VLink>
+              <!-- eslint-enable @intlify/vue-i18n/no-raw-text -->
```

Comment:
I _think_ this eslint-disable is superfluous. Double check me on that though!

```suggestion
              <VLink href="https://github.com/wordpress/openverse-frontend/">{{
                $t('about.planning.meta')
              }}</VLink>
```